code/mob/living/critter/adventure.dm

```diff
@@ -73,13 +73,7 @@
 				src.say( pick("Please! Help! I need help!", "Please...help me!", "Are you real? You're real! YOU'RE REAL", "Everything hurts! Everything hurts!", "Please, make the pain stop! MAKE IT STOP!") )
 
 	seek_target(var/range = 5)
```

Comment:
May as well remove this definition as it's just calling parent now. (same for all the other definitions that are just doing `. = ..()`)

Reply:
Ok, although some of these (for example, the seek_target at this line of code) have if checks after the initial parent call, so I don't think those can be removed.